lib/pangea/widgets/chat/message_toolbar.dart

```diff
@@ -61,80 +66,22 @@ class ToolbarDisplayController {
   void showToolbar(BuildContext context, {MessageMode? mode}) {
     // Close keyboard, if open
     FocusManager.instance.primaryFocus?.unfocus();
-    bool toolbarUp = true;
+    // Close emoji picker, if open
+    controller.showEmojiPicker = false;
```

Comment:
Good catch for this case! Looks like it's still possible to select a message while replying to a message and have the emoji picker not open, but the reply / forward bar does open. Do you think that could be resolved here?
<img width="1307" alt="Screenshot 2024-08-06 at 9 40 18 AM" src="https://github.com/user-attachments/assets/9ad9228c-e7b1-4aee-af2a-a47ed6b2e43a">